backend/app/api/routes/health.py

```
@@ -19,6 +20,34 @@
+        # Rebuild netloc without credentials
+        if parsed.hostname:
+            safe_netloc = parsed.hostname
+            if parsed.port:
+                safe_netloc = f"{safe_netloc}:{parsed.port}"
+        else:
+            safe_netloc = ""
+
+        # Reconstruct URL without credentials
+        sanitized = urlunparse((parsed.scheme, safe_netloc, parsed.path, "", "", ""))
+        return sanitized
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)